source/utilities.tex

```diff
@@ -3198,10 +3198,16 @@
   template<class T>
     class optional;                                                     // partially freestanding
 
+  // \ref{optional.optional.ref}, partial specialization of \tcode{optional} for lvalue reference types
   template<class T>
-    constexpr bool ranges::enable_view<optional<T>> = true;
+  class optional<T&>;                                                   // partially freestanding
+
+  template<class T>
+  constexpr bool ranges::enable_view<optional<T>> = true;
   template<class T>
-    constexpr auto format_kind<optional<T>> = range_format::disabled;
+  constexpr auto format_kind<optional<T>> = range_format::disabled;
+  template<class T>
+  constexpr bool ranges::enable_borrowed_range<optional<T&>> = true;
```

Comment:
For a non-defining declaration after a _template-head_, please indent by two spaces.
This applies predominantly to header files.  See the existing lines 3199 and 3202 for examples.

Reply:
Done.